tests/pkg_tinyusb_cdc_acm_stdio/usb_descriptors.c

```diff
@@ -0,0 +1,317 @@
+/*
```

Comment:
Regarding this, it makes little sense to redefine it again, since most of the parts are always the same. Furthermore, an application can't use other device classes if module `tinyusb_cdc_acm_stdio` is used.

That's why I'm working on a common `tinyusb_descriptors.c` file that uses the configuration and the Kconfig `CONFIG_*` symbols from the `usbus` to implement the descriptors. Once I have finished that work, we should be able to drop this file and it should be possible to use `tinyusb_stdio` together with other device class just by enabling the according `tinyusb_class_*` modules.

Reply:
If you already have a common `tinyusb_descriptors.c` in the works, I'll just wait for that :smiley: 

Moving this to XFA seems to be a bit tricky, you'll likely be faster with providing a generic solution as you already have experience with this package.